saleor/graphql/tax/schema.py

```diff
@@ -12,6 +12,12 @@
 from ..core.types import NonNullList
 from ..core.utils import from_global_id_or_error
 from .filters import TaxClassFilterInput, TaxConfigurationFilterInput
+from .mutations.tax_class_create import TaxClassCreate
+from .mutations.tax_class_delete import TaxClassDelete
+from .mutations.tax_class_update import TaxClassUpdate
+from .mutations.tax_configuration_update import TaxConfigurationUpdate
+from .mutations.tax_country_configuration_delete import TaxCountryConfigurationDelete
+from .mutations.tax_country_configuration_update import TaxCountryConfigurationUpdate
```

Comment:
Maybe we should create `__init__` in the mutations folder to have the option to import those mutations from one place. 
We have a similar solution in `/saleor/graphql/product/dataloaders/__init__.py`